app/controllers/CheckYourAnswersController.scala

```diff
@@ -108,12 +108,14 @@ class CheckYourAnswersController @Inject()(checkYourAnswersPage: CheckYourAnswer
       val penaltyType: String = PenaltyTypeEnum.withName(request.session.get(SessionKeys.appealType).get) match {
         case Late_Submission => "penaltyType.lateSubmission"
         case Late_Payment => "penaltyType.latePayment"
-        case Additional => "penaltyType.additional"
+        case Additional => "penaltyType.latePayment"
```

Comment:
This needs changing to `penaltyType.additional`
```suggestion
        case Additional => "penaltyType.additional"
```